common/djangoapps/enrollment/views.py

```diff
@@ -36,6 +36,28 @@
 log = logging.getLogger(__name__)
 
 
+class EmbargoMixin(object):
+    """
+    Mixins for embargo restrictions
+    """
+    def get_embargo_response(self, request, course_id, user):
+        """ Check whether any country access rules block the user from enrollment
+            We do this at the view level (rather than the Python API level)
+            because this check requires information about the HTTP request. """
+        redirect_url = embargo_api.redirect_if_blocked(
+            course_id, user=user, ip_address=get_ip(request), url=request.path)
+        if redirect_url:
+            return Response(
+                status=status.HTTP_403_FORBIDDEN,
+                data={
+                    "message": (
+                        u"Users from this location cannot access the course '{course_id}'."
+                    ).format(course_id=course_id),
+                    "user_message_url": request.build_absolute_uri(redirect_url)
+                }
+            )
+
+
```

Comment:
since `self` is never referenced in this function, it doesn't really need to be a mixin and could instead be a static function.  I think it should be defined as such in an `api.py` module instead.